config/manifests/kustomization.yaml

```diff
@@ -1,3 +1,7 @@
 resources:
 - ../default
 - ../samples
+
+patches:
+  - patches/serviceaccountname_in_deployment.yaml
```

Comment:
Just confirming that this makes sense to do in the case of Helm deployments as well? Any backcompat concerns?

Reply:
No, this isn't used in the helms chart - it's only used to produce the yaml that's fed into the operator bundle. Although it's pretty confusing that this directory is called config/manifests - it would be clearer to call it `config/operator-hub`. I'll rename it.

Reply:
Oh weird - for some reason it also needs to be passed the `--kustomize-dir` if it's not `config/manifests`, even though the content of that is what's sent on stdin.